ddl/constant.go

```diff
@@ -38,38 +38,34 @@ const (
 	HistoryTableID = meta.MaxInt48 - 3
 	// MDLTableID is the table ID of `tidb_mdl_info`.
 	MDLTableID = meta.MaxInt48 - 4
-	// BackfillTableID is the table ID of `tidb_ddl_backfill`.
-	BackfillTableID = meta.MaxInt48 - 5
-	// BackfillHistoryTableID is the table ID of `tidb_ddl_backfill_history`.
-	BackfillHistoryTableID = meta.MaxInt48 - 6
+	// BackgroundSubtasksTableID is the table ID of `tidb_background_subtasks`.
+	BackgroundSubtasksTableID = meta.MaxInt48 - 5
+	// BackgroundSubtasksHistoryTableID is the table ID of `tidb_background_subtasks_history`.
+	BackgroundSubtasksHistoryTableID = meta.MaxInt48 - 6
 
 	// JobTableSQL is the CREATE TABLE SQL of `tidb_ddl_job`.
 	JobTableSQL = "create table " + JobTable + "(job_id bigint not null, reorg int, schema_ids text(65535), table_ids text(65535), job_meta longblob, type int, processing int, primary key(job_id))"
 	// ReorgTableSQL is the CREATE TABLE SQL of `tidb_ddl_reorg`.
 	ReorgTableSQL = "create table " + ReorgTable + "(job_id bigint not null, ele_id bigint, ele_type blob, start_key blob, end_key blob, physical_id bigint, reorg_meta longblob, unique key(job_id, ele_id, ele_type(20)))"
 	// HistoryTableSQL is the CREATE TABLE SQL of `tidb_ddl_history`.
 	HistoryTableSQL = "create table " + HistoryTable + "(job_id bigint not null, job_meta longblob, db_name char(64), table_name char(64), schema_ids text(65535), table_ids text(65535), create_time datetime, primary key(job_id))"
-	// BackfillTableSQL is the CREATE TABLE SQL of `tidb_ddl_backfill`.
-	BackfillTableSQL = "create table " + BackfillTable + `(
-		id bigint not null,
-		ddl_job_id bigint not null,
-		ele_id bigint not null,
-		ele_key blob,
-		store_id bigint,
+	// BackgroundSubtasksTableSQL is the CREATE TABLE SQL of `tidb_background_subtasks`.
+	BackgroundSubtasksTableSQL = "create table " + BackgroundSubtasksTable + `(
+		id bigint not null auto_increment primary key,
+		namespace varchar(256),
+		task_key varchar(256),
+		ddl_physical_tid bigint(20),
 		type int,
-		exec_id blob default null,
-		exec_lease timestamp,
-		state int,
-		curr_key blob,
-		start_key blob,
-		end_key blob,
-		start_ts bigint,
-		finish_ts bigint,
-		row_count bigint,
-		backfill_meta longblob,
-		unique key(ddl_job_id, ele_id, ele_key(20), id))`
-	// BackfillHistoryTableSQL is the CREATE TABLE SQL of `tidb_ddl_backfill_history`.
-	BackfillHistoryTableSQL = "create table " + BackfillHistoryTable + `(
+		exec_id varchar(256) not null,
```

Comment:
I think this can be nil, the first insert, it's is nil.

Reply:
Done